db/seeds.rb

```diff
@@ -14,9 +17,13 @@
 fourth_post = Post.create(author_id: first_user.id, title: 'Hello!', text: 'This is my fourth post')
 
 # Comments
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hi Tom!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hi Tom!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hi!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hello there')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hello there!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hello')
+Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')
+Comment.create(post_id: first_post.id, author_id: second_user.id, text: 'Hi Tom!')
+Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Great Post')
+Comment.create(post_id: second_post.id, author_id: fourth_user.id, text: 'Nice Content, Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')
+Comment.create(post_id: third_post.id, author_id: third_user.id, text: 'Hello there!')
```

Comment:
Please add `user_id` instead of `author_id` when you create comments and posts in `seeds.rb` file as there is an error when the user runs `rails db:seed`.